nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFileTransfer.java

```diff
@@ -138,4 +144,32 @@ protected boolean isListingResetNecessary(final PropertyDescriptor property) {
     protected abstract FileTransfer getFileTransfer(final ProcessContext context);
 
     protected abstract String getProtocolName();
+
+    protected void validateAdjustedTimeWindow(ValidationContext validationContext, Collection<ValidationResult> results) {
+        if (
+            BY_ADJUSTED_TIME_WINDOW.getValue().equals(validationContext.getProperty(FILE_TRANSFER_LISTING_STRATEGY).getValue())
+            &&
+            !validationContext.getProperty(TIME_ADJUSTMENT).isSet()
+        ) {
+            results.add(new ValidationResult.Builder()
+                    .explanation("'" + TIME_ADJUSTMENT.getDisplayName() + "' has to be set when '"
+                        + FILE_TRANSFER_LISTING_STRATEGY.getDisplayName() + "' is set to '" + BY_ADJUSTED_TIME_WINDOW.getValue() + "'")
```

Comment:
`BY_ADJUSTED_TIME_WINDOW.getDisplayName()` should be used instead of `getValue()`
Also in line 169.

Reply:
Okay